merged da01e5fab9be9865db1aac203e574e0edbfd6584
"fixed compatibility with LibreSSL >= 3.5.0"
fixes https://github.com/OpenSC/OpenSC/issues/2664

Index: src/libopensc/card-iasecc.c
--- src/libopensc/card-iasecc.c.orig
+++ src/libopensc/card-iasecc.c
@@ -45,10 +45,6 @@
  * LibreSSL
  */
 
-#if defined(LIBRESSL_VERSION_NUMBER)
-# define  EVP_MD_CTX_md_data(x)  (x->md_data)
-#endif
-
 #include "internal.h"
 #include "asn1.h"
 #include "cardctl.h"
